### ISSUE TITLE
Problem showing inventory items

### ISSUE THREAD

**Issue by johntor** [DarkflameUniverse/NexusDashboard#45]
There is a problem displaying inventory items
![LUServer-2022-06-21-22-46-16](https://user-images.githubusercontent.com/478884/174885305-d5242b0b-65a9-4421-892d-24d1dd3683af.png)

As I can understand there is a problem converting dds files to png. All caches files (/app/cache)  although that are named .png after viewing as hex the header is DDS.

I had no problem/error while installing imagemagic.

I use linux Mint on Windows VM.

**Comment by johntor** [comment]
Sorry the problem was in imagemagic. I built it from current repository which is not working.

**Comment by aronwk-aaron** [comment]
> Sorry the problem was in imagemagic. I built it from current repository which is not working.

unsure what you mean by "from current repository which is not working", is there an issue?

**Comment by johntor** [comment]
Imagemagick if you clone/build/install there is a problem with dss. now my inventory looks great!!!
![LUServer-2022-06-21-23-41-11](https://user-images.githubusercontent.com/478884/174893725-0d5ee384-bd89-44c8-bbc3-d06cde6b1a99.png)
I downloaded an already build (potrable) copy. I placed it on /usr/local/bin. Emptied cache directory. Rerun server!

Great job [aronwk-aaron](https://github.com/aronwk-aaron)!!!

I wish I could edit inventor too!!!

Keep on!!!
